### ISSUE TITLE
Fix indentation rules for multiple classes and media queries in LESS

### ISSUE THREAD

**Issue by adinafxv** [visionappscz/stylelint-config-visionapps#8]
Now the rules demand this type of indentation for multiple classes & media queries:
```
.class-1,
    .class-2,
    .class-3 {
    @media only screen and (min-width: @screen-lg) {
        border: 1px solid #fff;
    }
}
```
And:
```
@media only screen and (min-width: @screen-lg) {
    .class-1,
        .class-2,
        .class-3 {
        border: 1px solid #fff;
    }
}
```
Please fix the rules so linter accepts this indentation:
```
.class-1,
.class-2,
.class-3 {
    @media only screen and (min-width: @screen-lg) {
        border: 1px solid #fff;
    }
}
```
And:
```
@media only screen and (min-width: @screen-lg) {
    .class-1,
    .class-2,
    .class-3 {
        border: 1px solid #fff;
    }
}
```

**Comment by adamkudrna** [comment]
This is probably caused by bug in Stylelint: https://github.com/stylelint/stylelint/issues/2791

Also happens with this code:

```css
.class1 input[type="text"],
.class2 input[type="email"] {
    font-size: inherit;
}
```

**Comment by adamkudrna** [comment]
Should be fixed in https://github.com/shellscape/postcss-less/issues/96.

**Comment by crishpeen** [comment]
Fixed with #17
